### DISCUSSION TITLE
why the filler-removing, and  silence-removing from audio makes my transcription better?

### DISCUSSION THREAD

**Discussion by HYUNHONOH98** [openai/whisper#607] [Q&A]
Hi open.ai! 

**First of all, thank you guys for making this tremendous transcribe model!!!**

I'm currently using whisper to transcribe the EFL's english-conversation audios.
However, I saw some audios that whisper's transcription quality goes bad. 

- ** ex: some words are not transcribed or mis-transcribed / words or sentences after silence are repeated themselves / no-punctuation mode occurs even when I used `initial_prompt = "~~"` method.**

Through case studies, I thought the problem is caused not only when there are a lot of filler-word spoken by EFL, but also a lot of silences on the audio.

So I removed some parts of audio that contain fillers and silences.
Then magically I could find out that the transcription has done perfectly.

these are few trials' options that I have done.

default : used **whisper - large model.**
1. using greedy decoding
- silence removed only - A
- filler removed + silence removed - B 

2. using beam search (default of python-executor)
- silence removed only - C
- ** filler removed + silence removed - D**

In my experience, case D showed the best transcription quality (which seems obviously better than A or B..)

**Comment by FurkanGozukara** [comment]
unfortunately it is due to way model is generated

i hope we get whisper 2 in future

**Reply by HYUNHONOH98** [reply]
@FurkanGozukara
Since the paper stated that they trained the only-silence audio data, I thought the silence would not affect the quality of transcription... 

Does the model structure of whisper has limit itself? 😢
